python/shaped_bloom_filter/exceptions.py

```diff
@@ -0,0 +1,6 @@
+class BloomFilterIncorrectConstructorValues(Exception):
+    pass
+
+
+class BloomFilterIncorrectInputException(Exception):
+    pass
```

Comment:
what are these for? and should we use the shaped base exception `MagnusException`?